<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>129</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>129</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>129</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/resources.qrc">
    <normaloff>:/Downloader/settings.svg</normaloff>:/Downloader/settings.svg</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLineEdit" name="m_curlLocation">
       <property name="toolTip">
        <string>Location of the curl binary</string>
       </property>
       <property name="placeholderText">
        <string>Enter location of the curl executable.</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="toolTip">
        <string>Location to store downloaded files.</string>
       </property>
       <property name="text">
        <string>Downloads folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="toolTip">
        <string>Location of the curl binary</string>
       </property>
       <property name="text">
        <string>curl executable location</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="toolTip">
        <string>Wait time between requests, in seconds.</string>
       </property>
       <property name="text">
        <string>Seconds between requests</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="m_waitSpinbox">
       <property name="toolTip">
        <string>Wait time between requests, in seconds.</string>
       </property>
       <property name="suffix">
        <string> seconds</string>
       </property>
       <property name="minimum">
        <number>5</number>
       </property>
       <property name="maximum">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="m_DownloadFolder">
       <property name="toolTip">
        <string>Location to store downloaded files.</string>
       </property>
       <property name="placeholderText">
        <string>Enter location of the downloads folder.</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="m_curlButton">
       <property name="toolTip">
        <string>Select the curl executable.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/resources.qrc">
         <normaloff>:/Downloader/folder.svg</normaloff>:/Downloader/folder.svg</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="m_downloadsButton">
       <property name="toolTip">
        <string>Select the download folder.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="resources/resources.qrc">
         <normaloff>:/Downloader/folder.svg</normaloff>:/Downloader/folder.svg</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>m_curlLocation</tabstop>
  <tabstop>m_DownloadFolder</tabstop>
  <tabstop>m_waitSpinbox</tabstop>
  <tabstop>m_curlButton</tabstop>
  <tabstop>m_downloadsButton</tabstop>
 </tabstops>
 <resources>
  <include location="resources/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigurationDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
